src/D2L.Bmx/Okta/Models/AuthenticateResponse.cs

```diff
@@ -8,7 +9,7 @@ internal record AuthenticateResponseInital(
 	string? StateToken,
 	string? SessionToken,
 	[property: JsonPropertyName( "_embedded" )]
-	AuthenticateResponseEmbeddedInitial Embedded
+	AuthenticateResponseEmbeddedInitial? Embedded
```

Comment:
there's no guarantee that this would exist. similarly here:
https://github.com/Brightspace/bmx/pull/307/files#diff-3da8e534d020fe4f4d0a2b1e6853e57fb85322b1a3a7c39e274c30acac0820b8L22-R23